pinot-common/src/test/java/org/apache/pinot/common/function/FunctionDefinitionRegistryTest.java

```diff
@@ -47,10 +47,8 @@ public class FunctionDefinitionRegistryTest {
       // Scalar function
       "scalar",
       // Functions without scalar function counterpart as of now
-      // TODO: Revisit this list when we add scalar function for these functions
-      "arraylength", "arrayaverage", "arraymin", "arraymax", "arraysum",
-      "valuein", "mapvalue", "inidset", "lookup", "groovy",
-      "timeconvert", "datetimeconvert", "jsonextractscalar", "jsonextractkey", "clpdecode",
+      "arraylength", "arrayaverage", "arraymin", "arraymax", "arraysum", "clpdecode", "groovy",
```

Comment:
other than the ignored list here, all other functions (including their alternative naming are registered in FunctionRegistry (e.g. has correspondent in ScalarFunction impl)